README.md

```diff
@@ -57,6 +57,57 @@ Install npm dependencies and start it up:
 
 This will start a local development server on port 5000 (http://localhost:5000).
 
+## Adding a project
+
+Each of the projects is a file in the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder - to add a new one, create a new file named after the project, ending in `.yaml`. Ensure all spaces and special
+characters are replaced with `-`. Make sure too that the project hasn't already been inside the [data](https://github.com/mozilla-frontend-infra/codetribute/tree/master/src/data)
+folder
+
+Codetribute can read from both GitHub and Bugzilla. To add a new entry to the site, create a file `<project-name>.yml` 
+in `src/data` using the [template example](#template-example) as the initial setup. For inspiration, check out the 
+[Taskcluster](https://github.com/mozilla-frontend-infra/codetribute/blob/master/src/data/taskcluster.yaml) yaml file.
+
+### Template Example
+
+```yaml
+name: <Project Name>
+summary: A short summary of the project
+introduction: |
+  ## About <Project Name>
+
+  ...
+
+  ## Who Works on <Project Name>?
+
+  ...
+
+  ## How Do I Get Started?
+
+  ...
+
+  ### How Do I Write the Code?
+
+  ...
```

Comment:
(summary of the development and patch-submission process -- pull requests? patches on bugzilla? tests?)